src/platform/w3dengine/client/win32mouse.h

```diff
@@ -35,10 +35,12 @@ class Win32Mouse : public Mouse
     virtual ~Win32Mouse() { g_theWin32Mouse = nullptr; };
 
     virtual void Init() override;
+    virtual void Reset() override { Mouse::Reset(); }
```

Comment:
? seems a bit odd to implement a function that is defined by default. Same goes for Update.